### PR TITLE
5253 metrics block ui

### LINKED ISSUE #5253: Display Make Data Count use and citation metrics
After the backend for Make Data Count support is implemented in #4821, we'll need to display the metrics on the appropriate pages.

### PR DESCRIPTION
**Pull request by pdurbin**
Closes #5253

Related to #1969

### CODE REVIEW

**Comment by mheppler** [comment]
> Line 411: "DATASET CITATIONS LINK TITLE HERE"
> needs to be bundle-ized with a real title?

Fixed. Removed the title attribute from the link.

File: src/main/java/propertyFiles/Bundle.properties [hunk 1]
```diff
@@ -1211,11 +1211,22 @@ dataset.exportBtn.itemLabel.oai_ore=OAI_ORE
...
+metrics.file.tip.default=Metrics for this individual file.
+metrics.file.tip.makedatacount=Individual file downloads are tracked but this is not part of the Make Data Count standard.
+metrics.file.downloads.tip=Total downloads of this file.
+metrics.views={0, choice, 0#Views|1#View|2#Views}
 metrics.downloads={0, choice, 0#Downloads|1#Download|2#Downloads}
-metrics.citations=Citations
-metrics.shares=Shares
+metrics.citations={0, choice, 0#Citations|1#Citation|2#Citations}
+metrics.citations.dialog.header=Dataset Citations
+metrics.citations.dialog.help=Citations for this dataset are retrieved from <a href="https://www.crossref.org" title="Crossref" target="_blank">Crossref</a> via DataCite using Make Data Count standards. For more information about dataset metrics, please refer to the <a href="{0}/{1}/user/dataset-management.html#dataset-metrics-user" title="Dataset Metrics - Dataverse User Guide" target="_blank">User Guide</a>.
```
Comment (mheppler) [inline src/main/java/propertyFiles/Bundle.properties]: @pdurbin I would recommend that the UI is not the appropriate place for an external link to find more information, but instead put this link in the guides, which we already link to in the next sentence.

Comment (pdurbin) [reply src/main/java/propertyFiles/Bundle.properties]: @mheppler thanks (especially for adding the links to the guides!). @djbrooke is going to put the finishing touches on this. (There's a new "to do" column in the spreadsheet.)

As of my commit, here's how the popup looks with two links:

Screen Shot 2019-07-22 at 3 27 32 PM

I believe we're going for slightly shorter wording, like this:

Screen Shot 2019-07-22 at 3 27 46 PM

@TaniaSchlatter will be reviewing all this. (I'm continuing to build and push the latest to http://ec2-3-86-35-164.compute-1.amazonaws.com/

**Comment by pdurbin** [comment]
@sekmiller I got settingsWrapper working in 0b4d21f if you'd like to take a look.
